cmd/kyma/apply/function/function.go

```diff
@@ -0,0 +1,238 @@
+package function
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/kyma-incubator/hydroform/function/pkg/client"
+	"github.com/kyma-incubator/hydroform/function/pkg/manager"
+	"github.com/kyma-incubator/hydroform/function/pkg/operator"
+	resources "github.com/kyma-incubator/hydroform/function/pkg/resources/unstructured"
+	"github.com/kyma-incubator/hydroform/function/pkg/workspace"
+	"github.com/kyma-project/cli/internal/cli"
+	"github.com/kyma-project/cli/internal/kube"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"os"
+)
+
+type command struct {
+	opts *Options
+	cli.Command
+}
+
+//NewCmd creates a new apply command
+func NewCmd(o *Options) *cobra.Command {
+	c := command{
+		opts:    o,
+		Command: cli.Command{Options: o.Options},
+	}
+	cmd := &cobra.Command{
+		Use:   "function",
+		Short: "Applies local resources for your Function to the Kyma cluster.",
+		Long: `Use this command to apply the local sources of your Function's code and dependencies to the Kyma cluster. 
+Use the flags to specify the desired location for the source files or run the command to validate and print the output resources.`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return c.Run()
+		},
+	}
+
+	cmd.Flags().StringVarP(&o.Filename, "filename", "f", "", `Full path to the config file.`)
+	cmd.Flags().BoolVar(&o.DryRun, "dry-run", false, `Validated list of objects to be created from sources.`)
+	cmd.Flags().Var(&o.OnError, "onerror", `Flag used to define the Kyma CLI's reaction to an error when applying resources to the cluster. Use one of these options: 
+- nothing
+- purge`)
+	cmd.Flags().VarP(&o.Output, "output", "o", `Flag used to define the command output format. Use one of these options:
+- text
+- json
+- yaml
+- none`)
+
+	return cmd
+}
+
+func (c *command) Run() error {
+	if c.opts.Filename == "" {
+		c.opts.Filename = defaultFilename()
+	}
+
+	file, err := os.Open(c.opts.Filename)
+	if err != nil {
+		return err
+	}
+
+	// Load project configuration
+	var configuration workspace.Cfg
+	if err := yaml.NewDecoder(file).Decode(&configuration); err != nil {
+		return errors.Wrap(err, "Could not decode the configuration file")
+	}
+
+	if c.K8s, err = kube.NewFromConfig("", c.KubeconfigPath); err != nil {
+		return errors.Wrap(err, "Could not initialize the Kubernetes client. Make sure your kubeconfig is valid")
```

Comment:
Comma is also not required here?

Reply:
Yes, because the `Wrap` method will merge these two strings (message and an error message) with `:` separator. So the final message will look like `... valid: ...`